content/en/docs/components/katib/overview.md

```diff
@@ -87,83 +86,82 @@ part of the form for submitting a NAS job from the Katib UI:
 
 ## The Katib project
 
-Katib is a Kubernetes-based system for hyperparameter tuning and neural 
-architecture search. Katib supports a number of ML frameworks, including 
+Katib is a Kubernetes-based system for hyperparameter tuning and neural
+architecture search. Katib supports a number of ML frameworks, including
 TensorFlow, MXNet, PyTorch, XGBoost, and others.
 
-The [Katib project](https://github.com/kubeflow/katib) is open source. 
+The [Katib project](https://github.com/kubeflow/katib) is open source.
 The [developer guide](https://github.com/kubeflow/katib/blob/master/docs/developer-guide.md)
 is a good starting point for developers who want to contribute to the project.
 
 ## Katib interfaces
 
 You can use the following interfaces to interact with Katib:
 
-* A web UI that you can use to submit experiments and to monitor your results.
-  See the [getting-started 
-  guide](/docs/components/hyperparameter-tuning/hyperparameter/#katib-ui)
+- A web UI that you can use to submit experiments and to monitor your results.
+  See the [getting-started
+  guide](/docs/components/katib/hyperparameter/#katib-ui)
   for information on how to access the UI.
   The Katib home page within Kubeflow looks like this:
 
-    <img src="/docs/images/katib/katib-home.png"
+  <img src="/docs/images/katib/katib-home.png"
       alt="The Katib home page within the Kubeflow UI"
       class="mt-3 mb-3 border border-info rounded">
 
-* A REST API. See the [API reference on 
-  GitHub](https://github.com/kubeflow/katib/blob/master/pkg/apis/manager/v1alpha3/gen-doc/api.md).
+- A gRPC API. See the [API reference](/docs/reference/katib/v1beta1/katib).
 
-* Command-line interfaces (CLIs):
+- Command-line interfaces (CLIs):
 
-  * **Kfctl** is the Kubeflow CLI that you can use to install and configure 
-    Kubeflow. Read about kfctl in the guide to 
+  - **Kfctl** is the Kubeflow CLI that you can use to install and configure
```

Comment:
```suggestion
  - **kfctl** is the Kubeflow CLI that you can use to install and configure
```

(do we use it with a capitalized letter K? Not 100% sure)

Reply:
Agree, changed.